### PR TITLE
Initial version of API Center .NET SDK.

### PR DESCRIPTION
**Pull request by azaslonov**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

**Comment by azaslonov** [comment]
@ArthurMa1978, is there anything needs to be done on our side to get this PR going? Please advise.

**Comment by azaslonov** [comment]
@ArthurMa1978, gentle ping.

File: sdk/apicenter/Azure.ResourceManager.ApiCenter/README.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+# Microsoft Azure ApiCenter management client library for .NET
+
+**[Describe the service briefly first.]**
```
Comment (ArthurMa1978) [inline sdk/apicenter/Azure.ResourceManager.ApiCenter/README.md:3]: Please describe the service briefly

File: sdk/apicenter/Azure.ResourceManager.ApiCenter/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Release History
+
+## 1.0.0-beta.1 (Unreleased)
```
Comment (ArthurMa1978) [inline sdk/apicenter/Azure.ResourceManager.ApiCenter/CHANGELOG.md:3]: Please specify the release date here by replacing `Unreleased`, the format is `yyyy-mm-dd`

File: sdk/apicenter/Azure.ResourceManager.ApiCenter/src/Properties/AssemblyInfo.cs [hunk 3]
```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Runtime.CompilerServices;
+
+[assembly: Azure.Core.AzureResourceProviderNamespace("ApiCenter")]
```
Comment (ArthurMa1978) [inline sdk/apicenter/Azure.ResourceManager.ApiCenter/src/Properties/AssemblyInfo.cs:6]: The value here is `Microsoft.ApiCenter`

**Comment by ArthurMa1978** [comment]
@alzaslon please see my comments.

File: sdk/apicenter/Azure.ResourceManager.ApiCenter/README.md (hunk 1 above)
Comment (azaslonov) [reply sdk/apicenter/Azure.ResourceManager.ApiCenter/README.md]: Done.

File: sdk/apicenter/Azure.ResourceManager.ApiCenter/CHANGELOG.md (hunk 2 above)
Comment (azaslonov) [reply sdk/apicenter/Azure.ResourceManager.ApiCenter/CHANGELOG.md]: Done.

File: sdk/apicenter/Azure.ResourceManager.ApiCenter/src/Properties/AssemblyInfo.cs (hunk 3 above)
Comment (azaslonov) [reply sdk/apicenter/Azure.ResourceManager.ApiCenter/src/Properties/AssemblyInfo.cs]: Done.

**Comment by azaslonov** [comment]
@ArthurMa1978, resolved your comments. Thanks!

**Comment by azaslonov** [comment]
@ArthurMa1978, gentle ping.

**Review by ArthurMa1978 — APPROVED** [review]

**Pull request merged**